docs/user/openstack/customization.md

```diff
@@ -117,6 +119,66 @@ platform:
       clusterOSImage: my-rhcos
 ```
 
+## Additional Networks
+
+You can set additional networks for your machines by defining `additionalNetworks` parameter in the machine configuration. The parameter is a list of objects that contain a network `uuid`:
+
+```yaml
+additionalNetworks:
```

Comment:
UX feels a bit clunky, for example we require the `uuid` attribute for additionalNetworks, but not for `additionalSecurityGroups`, while in the end they're both lists of UUIDs.

What is the idea behind making the `additionalNetworks` items objects? Does it mean we can also use the `name` attribute to identify my networks? I find it a bit confusing.

I suggest you rename this attribute to `additionalNetworkIDs` and be it a list of strings.

Reply:
done